docs/index.rst

```diff
@@ -90,7 +90,7 @@ contacting support.
 +--------------------------------------------------+---------------+----------------+
 | :ref:`flask_login`                               | >= 0.6.2      | Yes [6]_       |
 +--------------------------------------------------+---------------+----------------+
-| :ref:`gevent`                                    | >= 1.5        | No             |
+| :ref:`gevent` (greenlet>=1.0)                    | >= 20.12      | No             |
```

Comment:
Changing the [corporate documentation](https://github.com/DataDog/documentation/blob/5894a774b43284706c10b5dcfcf248d1352eb527/content/en/tracing/trace_collection/compatibility/python.md) to match would reduce the possibility for confusion.

Reply:
https://github.com/DataDog/documentation/pull/22033